src/components/scene/reflection.js

```diff
@@ -32,6 +32,7 @@ module.exports.Component = register('reflection', {
     this.cubeCamera = new THREE.CubeCamera(0.1, 1000, this.cubeRenderTarget);
     this.lightingEstimationTexture = (new THREE.WebGLCubeRenderTarget(16)).texture;
     this.needsVREnvironmentUpdate = true;
+    this.probeLight = null;
```

Comment:
I prefer to declare the variable in init like this. Also I check for it in the test.

Reply:
Do we need to declare it?